src/couch/test/eunit/couch_file_tests.erl

```diff
@@ -108,9 +116,16 @@ should_write_and_read_term(Fd) ->
     {ok, Pos, _} = couch_file:append_term(Fd, foo),
     ?assertMatch({ok, foo}, couch_file:pread_term(Fd, Pos)).
 
+should_read_write_multiple_terms(Fd) ->
+    {ok, [{Pos1, _}, {Pos2, _}]} = couch_file:append_terms(Fd, [foo, bar]),
+    ?assertMatch({ok, [bar, foo]}, couch_file:pread_terms(Fd, [Pos2, Pos1])).
```

Comment:
Good idea to swap the order here.

Reply:
Yeah I had figured it should work in any order so wanted to test the reverse order as well